### PR TITLE
[PLAYER-5374] Fix to get position of marker at seek

### PR DESCRIPTION
**Pull request by marcosmx**
This branch allow the seek action on the scrubber bar along with markers.

### CODE REVIEW

File: js/components/scrubberBar.jsx [hunk 1]
```diff
@@ -263,6 +263,10 @@ class ScrubberBar extends React.Component {
         : event.nativeEvent.offsetX;
     }
 
+    if (event.target.className.match('oo-marker')) {
```
Comment (KseniiaSokolovaEpam) [inline js/components/scrubberBar.jsx]: Do we need to test it?

Comment (marcosmx) [reply js/components/scrubberBar.jsx]: I tried, but looking at the test scenarios of the scrubber bar I didn't find any elaborate test over the scrubber bar actions ( mouseDown, mouseUp)...the ones that are written only check the existence of the event and nothing more.

Comment (ooyala-amohnatkin) [reply js/components/scrubberBar.jsx]: what if `event.target` is a child of `.oo-marker`?

Comment (marcosmx) [reply js/components/scrubberBar.jsx]: `.oo-marker` it's not intended to have child elements, so in this case we will never get the event propagation from a different source.

**Pull request merged**